src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    }
+
+    if (action.type === 'removeProperties') {
+      for (const key of action.keysToRemove) {
+        delete state[`${key}`];
```

Comment:
```suggestion
        delete state[key];
```
You don't need to convert `key` to string, cause inside `[]` the value of variable will be autonatically converted to string.

Reply:
fixed